graspy/embed/omni.py

```diff
@@ -173,6 +180,14 @@ def fit(self, graphs, y=None):
                 )
                 warnings.warn(msg, UserWarning)
 
+        # Diag augment
```

Comment:
this looks identical to the MASE one? maybe we have a `_multigraph_augment_diagonal` method in `BaseEmbedMulti`?

Reply:
Makes sense. Done.